.github/workflows/build-and-push.yml

```diff
@@ -31,6 +31,18 @@ jobs:
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             tag: "fedora"
             image_name: "nginx-124"
+          - dockerfile: "1.24/Dockerfile.c8s"
+            registry_namespace: "sclorg"
+            quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
+            quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
+            tag: "c8s"
+            image_name: "nginx-124-micro-c8s"
+          - dockerfile: "1.24/Dockerfile.c9s"
+            registry_namespace: "sclorg"
+            quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
+            quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
+            tag: "c9s"
+            image_name: "nginx-124-micro-c9s"
```

Comment:
The `micro` part looks suspicious -- this is not a micro one, is it?

Reply:
Ha, now I see @zmiklank already suggests a change..

Reply:
I suggested the change long ago, meanwhile marked as resolved. I have unresolved it now.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/278/commits/67ecede3d2da94df21b4fb2c5b0e5fe0349ecdd7